src/stories/material/Material.tsx

```diff
@@ -1,28 +1,35 @@
+import clsx from "clsx";
+
 export type MaterialProps = {
   url: string;
   animate: boolean;
   size: "small" | "medium" | "large";
+  tint: "20" | "40" | "60" | "80" | "120";
 };
 
-const placeholderBg = [
-  "bg-identity-tint-120",
-  "bg-identity-tint-80",
-  "bg-identity-tint-60",
-  "bg-identity-tint-40",
-  "bg-identity-tint-20",
-];
-
 export const Material = (props: MaterialProps) => {
-  // If Material does not have an img, show a random bg-color for each card.
-  const bg = placeholderBg[(Math.random() * placeholderBg.length) | 0];
+  const { size, animate, url } = props;
+
+  const classes = {
+    wrapper: clsx(
+      `material material--${size}`,
+      { "bg-identity-tint-120": props.tint === "120" },
+      { "bg-identity-tint-80": props.tint === "80" },
+      { "bg-identity-tint-60": props.tint === "60" },
+      { "bg-identity-tint-40": props.tint === "40" },
+      { "bg-identity-tint-20": props.tint === "20" },
+      { material__animate: animate }
+    ),
+  };
+
```

Comment:
Could be simplified, I think. (Not tested):
```

const placeholderBg = {
  "120": "bg-identity-tint-120",
  "80": "bg-identity-tint-80",
  "60": "bg-identity-tint-60",
  "40": "bg-identity-tint-40",
  "20": "bg-identity-tint-20"
};

const classes = {
  wrapper: clsx(
    `material material--${size}`,
    placeholderBg[tint] || false,
    { material__animate: animate }
  ),
};
```

Reply:
That is very true :) optimized it.